src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/ConnectivityTest.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Data;
```

Comment:
Remove and sort all usings. A few will go away.

Reply:
done